lightapi/python/symbollightapi/connector/NemConnector.py

```diff
@@ -274,3 +274,24 @@ async def announce_transaction(self, transaction_payload):
 			raise NodeException(f'announce transaction failed {response}')
 
 	# endregion
+
+	# region try_wait_for_announced_transaction
+
+	async def try_wait_for_announced_transaction(self, transaction_hash, desired_status, timeout_settings):
+		"""Tries to wait for a previously announced transaction to transition to a desired status."""
+
+		if TransactionStatus.UNCONFIRMED == desired_status:
+			# announce would fail if a transaction is not unconfirmed
```

Comment:
I am assuming the status does not matter if the transaction is already confirmed?